### PR TITLE
feat: remove goerli testnets

### PR DESCRIPTION
**Pull request by paulbalaji**
resolves https://github.com/hyperlane-xyz/issues/issues/991

remove all goerli support 

- goerli
- arb goerli
- op goerli
- linea goerli
- polygon zkevm testnet

### CODE REVIEW

File: rust/chains/hyperlane-ethereum/src/mailbox.rs [hunk 1]
```diff
@@ -437,7 +437,7 @@ mod test {
             provider.clone(),
             &ContractLocator {
                 // An Arbitrum Nitro chain
-                domain: &HyperlaneDomain::Known(KnownHyperlaneDomain::ArbitrumGoerli),
+                domain: &HyperlaneDomain::Known(KnownHyperlaneDomain::PlumeTestnet),
```
Comment (paulbalaji) [inline rust/chains/hyperlane-ethereum/src/mailbox.rs:440]: assuming this is fine as they're both on nitro stack

File: rust/sealevel/environments/testnet4/multisig-ism-message-id/solanatestnet/hyperlane/multisig-config.json [hunk 2]
```diff
@@ -1,108 +1,79 @@
 {
```
Comment (paulbalaji) [inline rust/sealevel/environments/testnet4/multisig-ism-message-id/solanatestnet/hyperlane/multisig-config.json:1]: vscode fixing formatting

File: rust/sealevel/environments/testnet4/multisig-ism-message-id/eclipsetestnet/hyperlane/multisig-config.json
Comment (paulbalaji) [inline rust/sealevel/environments/testnet4/multisig-ism-message-id/eclipsetestnet/hyperlane/multisig-config.json:1]: vscode fixing formatting

File: typescript/infra/config/environments/testnet4/hooks.ts
Comment (paulbalaji) [inline typescript/infra/config/environments/testnet4/hooks.ts:1]: deleted as it only references op goerli and couldn't find it being used anywhere

File: typescript/infra/scripts/funding/fund-keys-from-deployer.ts [hunk 3]
```diff
@@ -65,29 +65,14 @@ const nativeBridges = {
...
+
+const L2Chains: ChainName[] = [Chains.optimism, Chains.arbitrum, Chains.base];
 
 const L2ToL1: ChainMap<ChainName> = {
-  optimismgoerli: 'goerli',
-  arbitrumgoerli: 'goerli',
   optimism: 'ethereum',
   arbitrum: 'ethereum',
   base: 'ethereum',
-  polygonzkevmtestnet: 'goerli',
```
Comment (paulbalaji) [inline typescript/infra/scripts/funding/fund-keys-from-deployer.ts:90]: as mentioned in description, we will no longer bridge L1->L2 for polygonzkevmtestnet. This means we have to manually top up funder there from now on

File: rust/chains/hyperlane-ethereum/src/mailbox.rs (hunk 1 above)
Comment (paulbalaji) [reply rust/chains/hyperlane-ethereum/src/mailbox.rs:440]: cc @tkporter @daniel-savu

File: typescript/infra/scripts/funding/fund-keys-from-deployer.ts (hunk 3 above)
Comment (paulbalaji) [reply typescript/infra/scripts/funding/fund-keys-from-deployer.ts:90]: cc @nambrot

**Review by jmrossy — APPROVED** [review]
I haven't reviewed the `/rust/*` changes but the infra and sdk changes lgtm

File: typescript/infra/scripts/funding/fund-keys-from-deployer.ts (hunk 3 above)
Comment (nambrot) [reply typescript/infra/scripts/funding/fund-keys-from-deployer.ts:90]: loooks like that one will also get removed, so tbh i'm fine with removing polygonzkevm testnet as well https://polygon.technology/blog/polygon-pos-and-polygon-zkevm-new-testnets-for-polygon-protocols

Comment (paulbalaji) [reply typescript/infra/scripts/funding/fund-keys-from-deployer.ts:90]: nice, I'll take that down too then

File: rust/chains/hyperlane-ethereum/src/mailbox.rs (hunk 1 above)
Comment (daniel-savu) [reply rust/chains/hyperlane-ethereum/src/mailbox.rs:440]: if the tests pass then it's fine, the `l2_gas_limit` would be `None` if this wasn't an arbitrum nitro chain ([here](https://github.com/hyperlane-xyz/hyperlane-monorepo/blob/dan/prep-queue-labels/rust/chains/hyperlane-ethereum/src/mailbox.rs#L358))

**Review by daniel-savu — APPROVED** [review]

**Review by tkporter — APPROVED** [review]
lgtm but I didn't do a deep pass

**Pull request merged**